File: drawabox.py
#!/usr/bin/python
import sys
import argparse

parser = argparse.ArgumentParser()
parser.add_argument("width", type=int, help="The width of the box.")
parser.add_argument("height", type=int, help="This will be the height of the box.")
args = parser.parse_args()

width = args.width
height = args.height

def box(width,height):
    print "*" * width
    for middle in range(height - 2):
        print "|" + ' ' * (width - 2) + "|"
    print "*" * width

box(width,height)

File: drawabox.rb
#!/usr/bin/env ruby

width = ARGV[0].to_i
height = ARGV[1].to_i

def box(x,y)
    puts "*" * x 
    0.upto(y-2) do 
    puts("|" + ' ' * (x- 2) + "|")
    end
  puts "*" * x 
end

box(width,height)
